demos/node-classification/rgcn/rgcn-aifb-node-classification-example.ipynb

```diff
@@ -0,0 +1,414 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stellargraph example: Relational Graph Convolutional Network (RGCN) on the AIFB relational dataset\n",
+    "\n",
+    "This example demonstrates how use an RGCN [1] on the AIFB dataset with stellargraph. This dataset can be downloaded [here](https://figshare.com/articles/AIFB_DataSet/745364). \n",
+    "\n",
+    "The following is a description of the dataset:\n",
+    "\n",
+    ">The AIFB dataset describes the AIFB research institute in terms of its staff, research group, and publications. It contains ~8k entities, ~29k edges, and 45 different relationships or edge types. In (Bloehdorn et al 2007) the dataset was first used to predict the affiliation (i.e., research group) for people in the dataset. The dataset contains 178 members of a research group with 5 different research groups.\n",
+    "\n",
+    "The goal is to predict which research group a researcher belongs to.\n",
+    "\n",
+    "[1] Modeling Relational Data with Graph Convolutional Networks. Thomas N. Kipf, Michael Schlichtkrull (2017). https://arxiv.org/pdf/1703.06103.pdf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we load the required libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from rdflib.extras.external_graph_libs import *\n",
+    "from rdflib import Graph, URIRef, Literal\n",
+    "\n",
+    "import networkx as nx\n",
+    "from networkx.classes.function import info\n",
+    "\n",
+    "import stellargraph as sg\n",
+    "from stellargraph.mapper.node_mappers import RelationalFullBatchNodeGenerator\n",
```

Comment:
Don't import from `node_mappers` : this file no longer exists. The imports should be:
```
from stellargraph.mapper import RelationalFullBatchNodeGenerator
```

To do this you'll need to add `RelationalFullBatchNodeGenerator` to the `__all__` variable at the top of the `mappers/full_batch_generators.py` file so they are inserted into the `mappers` namespace.

Reply:
done!